compose/cli/main.py

```diff
@@ -895,6 +895,7 @@ def up(self, options):
             --no-deps                  Don't start linked services.
             --force-recreate           Recreate containers even if their configuration
                                        and image haven't changed.
+            --always-recreate-deps     Recreate dependant containers.
```

Comment:
Let's use the "dependent" spelling instead, for consistency with https://github.com/docker/compose/blob/master/compose/config/sort_services.py#L34